pkg/cri/server/container_create_windows.go

```diff
@@ -51,9 +51,8 @@ func (c *criService) containerSpec(
 	extraMounts []*runtime.Mount,
 	ociRuntime config.Runtime,
 ) (*runtimespec.Spec, error) {
-	specOpts := []oci.SpecOpts{
-		customopts.WithProcessArgs(config, imageConfig),
-	}
+	var specOpts []oci.SpecOpts
+	specOpts = append(specOpts, customopts.WithProcessCommandLineOrArgs(config, imageConfig))
```

Comment:
Could you pls backport this to `sbserver` as well?

Reply:
The code refactoring seems to be very different in sbserver. Do we need to put windows specific functions under _windows.go there too?